### PR TITLE
:goal_net:(llm): undelegate error message not enought for fees 

### PR DESCRIPTION
**Pull request by LucasWerey**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [x] **Impact of the changes:** 
  - undelagation flow

### 📝 Description

Rework the UI of undelegate error on LLM so the user has a better message. For Ethereum, Solana, Cardano, Polkadot, Near, Injective, MultiverseX, Dydx, Osmosis, Tezos, Celo.

UI is not exactly the same as figma so I did it with the current version. 

Remaning todos : 
- [ ] Eth  (No account)
- [x] Solana
- [x] Cardano 
- [x] Polk
- [x] Near 
- [x] Inj
- [ ] MX (No account)
- [x] Dydx
- [x] Osmo
- [x] Tezos
- [x] Celo
- [x] Cosmos

Each screen is simulated changing fees inside coin-modules that's why we can't pay the fees even with more than usual fees

Each link redirects to the wanted screen if possible so user can buy/swap/receive

| COIN        | IMG         |
| ------------- | ------------- |
|**COSMOS FAMILY**|
|COSMOS|image|
|INJ|image|
|OSMOSIS|image|
|DYDX|image|
|**OTHERS**|
|NEAR|image|
|POLKADOT|image|
|CELO|image
|SOLANA|image|
|CARDANO|image|
|TEZOS|image|

### ❓ Context

- **JIRA or GitHub link**: [LIVE-11353](https://ledgerhq.atlassian.net/browse/LIVE-11353)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-11353]: https://ledgerhq.atlassian.net/browse/LIVE-11353?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/polkadot/UnbondFlow/01-Amount.tsx [hunk 1]
```diff
@@ -110,6 +112,9 @@ export default function PolkadotUnbondAmount({ navigation, route }: Props) {
   const error = amount.eq(0) || bridgePending ? null : getFirstStatusError(status, "errors");
   const warning = getFirstStatusError(status, "warnings");
   const hasErrors = hasStatusError(status);
+  const isUnbonding = transaction.mode === "unbond";
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/families/polkadot/UnbondFlow/01-Amount.tsx:115]: Do we need this as we are already in the unbound flow ? The file is in the `UnboundFlow` directory

File: apps/ledger-live-mobile/src/families/celo/RevokeFlow/VoteAmount.tsx [hunk 2]
```diff
@@ -96,6 +98,8 @@ export default function VoteAmount({ navigation, route }: Props) {
   const unit = useAccountUnit(account);
   const error = amount.eq(0) || bridgePending ? null : getFirstStatusError(status, "errors");
   const warning = getFirstStatusError(status, "warnings");
+  const isRevoking = transaction.mode === "revoke";
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/families/celo/RevokeFlow/VoteAmount.tsx:101]: Do we need this as we are already in the revoke flow ? The file is in the `RevokeFlow` directory

File: apps/ledger-live-mobile/src/families/shared/StakingErrors/NotEnoughFundFeesAlert/index.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  );
+
+  const onNavigate = useCallback(
+    (screenName: keyof typeof mapActionButtons, navigatorName: string, params?: object) => {
+      track("button_clicked", {
+        button: mapActionButtons[screenName],
+        asset: currency.name,
+        source: "UndelegateFlowScreen",
+      });
+      (navigation as StackNavigationProp<{ [key: string]: object | undefined }>).navigate(
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/families/shared/StakingErrors/NotEnoughFundFeesAlert/index.tsx:59]: Can we avoid using the `as` keyword ?
For example like this : 
Screenshot 2024-10-24 at 18 43 35

File: apps/ledger-live-mobile/src/families/celo/RevokeFlow/VoteAmount.tsx (hunk 2 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/families/celo/RevokeFlow/VoteAmount.tsx:101]: removed ✅

File: apps/ledger-live-mobile/src/families/polkadot/UnbondFlow/01-Amount.tsx (hunk 1 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/families/polkadot/UnbondFlow/01-Amount.tsx:115]: removed ✅

File: apps/ledger-live-mobile/src/families/shared/StakingErrors/NotEnoughFundFeesAlert/index.tsx (hunk 3 above)
Comment (LucasWerey) [reply apps/ledger-live-mobile/src/families/shared/StakingErrors/NotEnoughFundFeesAlert/index.tsx]: Done ✅

**Review by cgrellard-ledger — APPROVED** [review]

**Review by Wozacosta — APPROVED** [review]

**Pull request merged**